packages/toolpad-app/src/runtime/ToolpadApp.spec.tsx

```diff
@@ -2,13 +2,16 @@ import * as React from 'react';
 import { render, waitFor as waitForOrig, screen, fireEvent, act } from '@testing-library/react';
 import '@testing-library/jest-dom';
 import { LiveBindings, RuntimeEvents } from '@mui/toolpad-core';
+import failOnConsole from 'jest-fail-on-console';
 import ToolpadApp from './ToolpadApp';
 import * as appDom from '../appDom';
 import createRuntimeState from '../createRuntimeState';
 import { bridge } from '../canvas/ToolpadBridge';
 import { BridgeContext } from '../canvas/BridgeContext';
 import loadComponents from './loadDomComponents';
 
+failOnConsole();
```

Comment:
Would it be possible to set this globally for any spec file? 
This library seems to recommend using the file for `setupFilesAfterEnv` for that.

Reply:
Sure 👍